ospf6d/ospf6_lsa.h

```diff
@@ -18,6 +18,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#include "lib/json.h"
```

Comment:
I have added this header here, because the function declarations below use the json_object data structure.